code/api/src/modules/user/resolvers.js

```diff
@@ -77,3 +77,47 @@ export async function remove(parentValue, { id }) {
 export async function getGenders() {
   return Object.values(params.user.gender)
 }
+
+// Update user
+export async function update(parentValue, { id, style }) {
+  let styleArray = style.split(', ')
+  let styleCount = {}
+
+  styleArray.forEach((val) => {
+    if (!styleCount[val]) {
+      styleCount[val] = 1
+    }
+    else {
+      styleCount[val]++
+    }
+  })
+
+  let styleKeys = Object.keys(styleCount);
+
+  let styleKeysSorted = styleKeys.sort(function (a, b) { return styleCount[a] - styleCount[b] }).reverse();
```

Comment:
If styleCount[a] & [b] are reversed in the return statement, can the following .reverse() be removed?